README.md

```diff
@@ -1459,18 +1432,117 @@ this in mind, an implementation in IPLD/IPFS is favored:
     "share": "1, 2, 3, ..., 100",
     "validFrom": { "/": "<hash pointing to RDF-Schema of Date" },
     "validTo": { "/": "<hash pointing to RDF-Schema of Date" },
-    "manifestation": { "/": "<hash pointing to the a Manifestation>" },
-    "license": { "/": "<hash pointing to the license>" }
+    "source": { "/": "<hash pointing to a Copyright>" },
```

Comment:
Chose to use `source` instead of `allowedBy` as it maps nicely to some other ontologies and (may) sound clearer. Will force changes in pycoalaip and coalaip-http-api.